alterconfigs.go

```diff
@@ -0,0 +1,106 @@
+package kafka
```

Comment:
Do you think you could include the tests added in https://github.com/segmentio/kafka-go/pull/470?

Reply:
actually, here are too many different between 0.4 and master, so i don't think the UT of the master branch is not too much sense.

Reply:
I see, maybe we can add some new simple tests, just to make sure the code runs and doesn't crash for each of the 3 API requests

Maybe something like:
- create config
- AlterConfigs (try to exercise as many code paths as is easy to test)
- DescribeConfigs and see if config is as expected

And for CreatePartition:
- CreatePartition (try to exercise various options)
- query to see partition is created as expected